lib/notifications/display.dart

```diff
@@ -103,8 +126,15 @@ class NotificationDisplayManager {
       FcmMessageDmRecipient() =>
         data.senderFullName,
     };
-    final groupKey = _groupKey(data);
-    final conversationKey = _conversationKey(data, groupKey);
+
+    messagingStyle.messages?.add(MessagingStyleMessage(
+      text: data.content,
+      timestampMs: data.time * 1000,
+      person: Person(
+        key: data.senderId.toString(),
```

Comment:
Should we add the org URL to the key? There could be users on different realms with the same ID.